### PR TITLE
GET endpoints for trip invitations

### PR DESCRIPTION
**Pull request by connie-code**

### CODE REVIEW

File: database.sql [hunk 1]
```diff
@@ -20,17 +20,17 @@ CREATE TABLE userTable(
...
 );
 
 -- Invitations table
 CREATE TABLE invitations(
-  hostID INT UNIQUE,
-  userID INT UNIQUE,
-  tripID INT UNIQUE,
+  hostID INT,
+  userID INT,
+  tripID INT,
```
Comment (Quetourah) [inline database.sql]: shouldn't a tripID be unique?

File: server.js [hunk 2]
```diff
@@ -60,6 +60,22 @@ app.post('/login', passport.authenticate('local'), (req, res) => {
 	res.json(user);
 });
```
Comment (Quetourah) [inline server.js]: Is this so we can display trip invitations to a user?

File: database.sql (hunk 1 above)
Comment (kcode20) [reply database.sql]: No, these should not be unique, we can have multiple users invited to the same trip.

File: server.js (hunk 2 above)
Comment (kcode20) [reply server.js]: Yes, this is. We want to be able to return the invitation information to the front-end so guest can be informed of the open invitations so they can accept and decline invitations.

File: server.js [hunk 3]
```diff
@@ -60,6 +60,22 @@ app.post('/login', passport.authenticate('local'), (req, res) => {
...
 
+app.get('/invitations/:userid', (req, res) =>{
+	pool.query(
+		'SELECT * FROM invitations NATURAL JOIN trips NATURAL JOIN usertable WHERE userid=$1',
```
Comment (kcode20) [inline server.js]: I am wondering if we should pass everything or select only certain columns. We can revisit this after the FE portion.

File: database.sql (hunk 1 above)
Comment (Quetourah) [reply database.sql]: ah. i see.

File: server.js (hunk 3 above)
Comment (Quetourah) [reply server.js]: I think we can circle back on we see what we want to use in the front end. However, we most likely don't need everything.

**Pull request merged**